### PR TITLE
Proof for ~opabssi, catching up STS formatting

### PR DESCRIPTION
**Pull request by tirix**
- Proof for Peter's `~opabssi` theorem discussed in #2502 without the `Rel A` hypothesis
- Updates to the MathML/STS formatting.

### CODE REVIEW

**Comment by benjub** [comment]
Can you see if you can also prove the version of opabid2 I suggested at https://github.com/metamath/set.mm/pull/2502#discussion_r809969982 ?

File: set.mm [hunk 1]
```diff
@@ -464135,6 +464135,17 @@ Class abstractions (a.k.a. class builders)
       FUMQABCTUAUBUCUDUE $.
   $}
 
+  ${
+    $d A x y z $.  $d ph z $.
+    opabssi.1 $e |- ( ph -> <. x , y >. e. A ) $.
+    $( Sufficient hypothesis for a collection of ordered pairs to be a subclass
+       of a relation.  (Contributed by Peter Mazsa, 21-Oct-2019.) $)
+    opabssi $p |- { <. x , y >. | ph } C_ A $=
```
Comment (mazsa) [inline set.mm:464143]: 1. I see, great:) 2. No way. I insist on this addition at least : (Revised by Thierry Arnoux, 18-Feb-2022.)

Comment (mazsa) [reply set.mm:464143]: @benjub do you think this "Revised by" is OK? I think @tirix thinks that you was the first who had the idea. But Thierry implemented it.

Comment (benjub) [reply set.mm:464143]: I think it's ok as is, but the credit could as well go to @tirix, that's not very important in this case.  The general view is that "ideas are cheap, implementation is hard", so credits should generally go to the person who does the formalization over the one who has the idea of doing it.

Comment (mazsa) [reply set.mm:464143]: @benjub I completely agree. @tirix please correct this.

Comment (mazsa) [reply set.mm:464143]: @tirix I mean please correct this in a way that you Thierry is the author.

**Comment by tirix** [comment]
> Can you see if you can also prove the version of `~ opabid2` I suggested at [#2502 (comment)](https://github.com/metamath/set.mm/pull/2502#discussion_r809969982) ?

See last commit: is that what you meant?

File: set.mm (hunk 1 above)
Comment (tirix) [reply set.mm:464143]: Ok, since it's my choice I kept the version with contributed/revised. 😁

**Comment by benjub** [comment]
> > Can you see if you can also prove the version of `~ opabid2` I suggested at [#2502 (comment)](https://github.com/metamath/set.mm/pull/2502#discussion_r809969982) ?
> 
> See last commit: is that what you meant?

Yes, that's the statement ! (although I was thinking of proving opabssi from opabid2ss instead of the other way around, but yours probably makes for a shorter total)

**Review by benjub — APPROVED** [review]

File: set.mm [hunk 2]
```diff
@@ -464135,6 +464135,26 @@ Class abstractions (a.k.a. class builders)
       FUMQABCTUAUBUCUDUE $.
   $}
 
+  ${
+    $d A x y z $.  $d ph z $.
+    opabssi.1 $e |- ( ph -> <. x , y >. e. A ) $.
+    $( Sufficient hypothesis for a collection of ordered pairs to be a subclass
```
Comment (benjub) [inline set.mm:464141]: "Sufficient condition", rather ?

**Review by avekens — APPROVED** [review]

File: set.mm (hunk 2 above)
Comment (tirix) [reply set.mm:464141]: Indeed! Fixed.

**Pull request merged**